cves/kernel/CVE-2014-3183.yml

```diff
@@ -75,7 +91,9 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
+
 bugs: []
```

Comment:
I think there would be a bug associated with this vulnerability as it is a software issue indicating that there exists a bug. I would look again and try to find any mention of a bug id in any of the sources that you used.

Reply:
I couldn't find any bug IDs either. The bug was found by a Project Zero member, and it was reported over (I presume) Linux kernel's security team email, not Bugzilla, so there should be no bug IDs.